crates/router/src/connector/forte/transformers.rs

```diff
@@ -252,14 +252,16 @@ impl<F, T>
         Ok(Self {
             status: enums::AttemptStatus::foreign_from((response_code, action)),
             response: Ok(types::PaymentsResponseData::TransactionResponse {
-                resource_id: types::ResponseId::ConnectorTransactionId(transaction_id.to_string()),
+                resource_id: types::ResponseId::ConnectorTransactionId(
+                    transaction_id.clone().to_string(),
+                ),
                 redirection_data: None,
                 mandate_reference: None,
                 connector_metadata: Some(serde_json::json!(ForteMeta {
                     auth_id: item.response.authorization_code,
                 })),
                 network_txn_id: None,
-                connector_response_reference_id: None,
+                connector_response_reference_id: Some(transaction_id.clone().to_string()),
```

Comment:
Same here and Please change it in other places as well .

Reply:
Thanks for the feedback @srujanchikke , that's good to know!

Reply:
Hi @AaronTJDev 
Please use following cargo commands - 
```
cargo check --all-features                                                                  
cargo clippy --all-features
```
ref - https://github.com/juspay/hyperswitch/blob/main/docs/CONTRIBUTING.md#cargo-commands, and fix the errors thrown by clippy for the checks to pass

Please run `cargo +nightly fmt` to fix failing formatting check